drivers/crypto/crypto_stm32.c

```diff
@@ -95,35 +114,21 @@ static int do_encrypt(struct cipher_ctx *ctx, uint8_t *in_buf, int in_len,
 	return 0;
 }
 
-static int do_decrypt(struct cipher_ctx *ctx, uint8_t *in_buf, int in_len,
-		      uint8_t *out_buf)
+#if !DT_HAS_COMPAT_STATUS_OKAY(st_stm32l4_aes)
```

Comment:
Nitpick: `!DT_HAS_COMPAT_STATUS_OKAY(st_stm32l4_aes)` is not actually required in this commit